src/rabbit_policy.erl

```diff
@@ -212,11 +212,11 @@ parse_set(Type, VHost, Name, Pattern, Definition, Priority, ApplyTo) ->
     end.
 
 parse_set0(Type, VHost, Name, Pattern, Defn, Priority, ApplyTo) ->
-    case rabbit_json:try_decode(Defn) of
+    case rabbit_json:try_decode(list_to_binary(Defn)) of
```

Comment:
In the new CLI all Elixir strings are binaries, so I don't think we can get a list here.